app/display/representation-javafx/src/main/java/org/csstudio/display/builder/representation/javafx/SVGHelper.java

```diff
@@ -39,6 +44,32 @@
  */
 public class SVGHelper {
 
+
+    /**
+     * Loads SVG file as an {@link Image}. The resolution of the generated image is determined by the
+     * width and height parameters. Consequently scaling to a much larger size will result in "pixelation".
+     * Client code is hence advised to reload the SVG resource using the new width and height when
+     * the container widget is resized.
+     *
+     * @param imageFileName Non-null, absolute path to SVG resource.
+     * @param width The wanted width of the image.
+     * @param height The wanted height of the image.
+     * @return A {@link Image} object if the input stream can be parsed and transcoded.
+     */
+    public static Image loadSVG(String imageFileName, double width, double height){
+        String cachedSVGFileName = imageFileName + "_" + width + "_" + height;
+        return ImageCache.cache(cachedSVGFileName, () ->
+        {
+            // Open the image from the stream created from the resource file.
+            try(InputStream inputStream = ModelResourceUtil.openResourceStream(imageFileName)){
+                return loadSVG(inputStream, width, height);
+            } catch ( Exception ex ) {
+                logger.log(Level.WARNING, "Failure loading image: {0} [{1}].", new Object[] { imageFileName, ex.getMessage() });
```

Comment:
Instead of
```
logger.log("Error: " + ex.getMessage());
```
use 
```
logger.log("Error", ex);
```

The former only logs the exception message, which might be null. The latter logs the message (if there is one), plus the stack trace, and the optional nested 'cause'.